subt_ign/src/LogChecker.cc

```diff
@@ -193,7 +193,7 @@ class LogChecker
       return 1;
 
     // Begin playback as fast as possible.
-    const auto handle = this->player->Start(std::chrono::seconds(1), false);
+    const auto handle = this->player->Start(std::chrono::seconds(1));
```

Comment:
just checking if the removal of second arg is intended? Looking at [ign-transport doc](https://github.com/ignitionrobotics/ign-transport/blob/ign-transport8/log/include/ignition/transport/log/Playback.hh#L96) the default is `true` which means it will no longer playback as fast as possible. If that's intended, we can just update the comment above this line.